README.md

```diff
@@ -414,6 +415,22 @@ Set up (the latest version of) [HAProxy](http://www.haproxy.org/) in Ubuntu syst
 * `haproxy_acl_files.{n}.dest`: [required]: The remote path of the file (e.g. `/etc/haproxy/acl/api.map`)
 * `haproxy_acl_files.{n}.content`: [default: `[]`]: The content (lines) of the file (e.g. `['v1.0 be_alpha', 'v1.1 be_bravo']`)
 
+* `haproxy_scripts_ssl_deploy_template`: [default: `usr/local/bin/haproxy-ssl-deploy.j2`]: Template to deploy SSL certificates after creation and renewal (for example by Letsencrypt)
```

Comment:
As this is Letsencrypt only, should we renamble `haproxy_scripts_` to `haproxy_letsencrypt_`?

Reply:
Although this could be used for other authorities (it only requires a source path with the certificate directories), we could make it specific. Do you prefer that?

Reply:
Yes, I'm looking into a "generic" OCSP script (for instance https://github.com/pierky/haproxy-ocsp-stapling-updater)